app/components/code-mirror.ts

```
@@ -49,7 +49,94 @@
+          }),
+          keymap.of(foldKeymap),
+        ]
+      : [],
+  lineSeparator: ({ lineSeparator: lineSeparatorText }) => (lineSeparatorText !== undefined ? [EditorState.lineSeparator.of(lineSeparatorText)] : []),
+  lineWrapping: ({ lineWrapping: enabled }) => (enabled ? [EditorView.lineWrapping] : []),
+  placeholder: ({ placeholder: placeholderText }) => (placeholderText !== undefined ? [placeholder(placeholderText)] : []),
+  readOnly: ({ readOnly: enabled }) => [EditorState.readOnly.of(!!enabled)],
+  rectangularSelection: ({ rectangularSelection: enabled }) => (enabled ? [rectangularSelection()] : []),
+  scrollPastEnd: ({ scrollPastEnd: enabled }) => (enabled ? [scrollPastEnd()] : []),
+  syntaxHighlighting: ({ syntaxHighlighting: enabled }) => (enabled ? [syntaxHighlighting(defaultHighlightStyle, { fallback: true })] : []),
+  tabSize: ({ tabSize: tabSize }) => (tabSize !== undefined ? [EditorState.tabSize.of(tabSize)] : []),
```

Biome

[error] Useless rename.

Safe fix: Remove the renaming.


(lint/complexity/noUselessRename)